### ISSUE TITLE
False errors in clion using async-std

### ISSUE THREAD

**Issue by tekjar** [intellij-rust/intellij-rust#4641]
CLion 2019.2.5
Build #CL-192.7142.39, built on October 29, 2019
Runtime version: 11.0.4+10-b304.77 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.15
GC: ParNew, ConcurrentMarkSweep
Memory: 1987M
Cores: 4
Registry: run.processes.with.pty=TRUE
Non-Bundled Plugins: IdeaVIM, org.intellij.plugins.markdown, org.toml.lang, org.rust.lang

## Problem description

Doesn't complete some crate's types

```rust
use async_std::net::{TcpListener, ToSocketAddrs, TcpStream};
use async_std::io::BufReader;


async fn connection_loop(stream: TcpStream) -> Result<(), Error> {
    let reader = BufReader::new(&stream); // 2
}
```

Clion shows all these as errors

**Comment by daharon** [comment]
Possible that this is a duplicate of #4614?

**Comment by Undin** [comment]
Duplicate of #4627
